### DISCUSSION TITLE
I hope that FTP and webdav can be used for storage

### DISCUSSION THREAD

**Discussion by Xun-X** [usememos/memos#3607] [Ideas]
I hope that FTP and webdav can be used for storage. Currently, there are too few storage methods.

**Comment by 855579762735** [comment]
@Xun-X, Looks like I completely missed this discussion about WebDAV support haha .. when you say WebDAV do you mean storing markdown files on a filesystem over WebDAV ? or do you mean to support the **actual standardized implementations of journals within CalDAV** as I outlined here;
https://github.com/orgs/usememos/discussions/4548
which I think would be far more powerful.
